ramls/schemas/jobExecution.json

```
@@ -0,0 +1,61 @@
+{
+  "$schema":"http://json-schema.org/draft-04/schema#",
+  "description":"Job Execution Schema",
+  "type":"object",
+  "properties":{
+    "id":{
+      "description":"Unique identifier",
+      "$ref":"../../ramls/raml-util/schemas/uuid.schema"
+    },
+    "hrId":{
+      "description":"Human readable id",
+      "type":"string"
+    },
+    "fileName":{
```

Going forward we will be having a list of files, probably makes sense to make this an array

---

Let's make it array

---

add file Id and filename
```suggestion
    "file"{
    "fileId": 
    "fileName":
    }
```